src/index.js

```diff
@@ -199,6 +199,14 @@ async function bundleMDX({
             varName: '_jsx_runtime',
             type: 'cjs',
           },
+          'react/jsx-dev-runtime': {
+            varName: '_jsx_runtime',
+            type: 'cjs'
+          },
+          'react/compiler-runtime': {
+            varName: '_react_compiler_runtime',
+            type: 'cjs'
+          }
```

Comment:
This isn't necessary for React 19 compat. But if you ever apply the React compiler, you need to externalize its runtime so I just went ahead and did it now to avoid future bugs.